src/playercards/CardsThatSealTokens.ttslua

```diff
@@ -0,0 +1,154 @@
+--[[ Library for cards that seal tokens
+This file is used to add sealing option to cards' context menu.
+Valid options (set before requiring this file):
+SHOW_READ_BAG       -- boolean
+SHOW_SINGLE_RELEASE -- boolean
+SHOW_MULTI_RELEASE  -- number (amount of tokens to release at once)
+VALID_TOKENS        -- table ([tokenName] = true)
+INVALID_TOKENS      -- table ([tokenName] = true)]]
```

Comment:
new required file for cards with sealing - this section contains the options that can be changed (by including them before using `require`)